fullscreen-basic/definition.json

```diff
@@ -362,7 +362,7 @@
         },
         {
           "name": "deeplink_url",
-          "default": "myapp://path/to/content or http://your.url",
+          "placeholder": "myapp://path/to/content or http://your.url",
```

Comment:
I don't think you should remove the default value, but instead keep both the default and the placeholder. You can change the default to be empty though, for deeplinks